Lib/fontTools/feaLib/builder.py

```diff
@@ -1046,23 +1047,28 @@ def add_mark_mark_pos(self, location, baseMarks, marks):
     def add_class_pair_pos(self, location, glyphclass1, value1,
                            glyphclass2, value2):
         lookup = self.get_lookup_(location, PairPosBuilder)
-        lookup.addClassPair(location, glyphclass1, value1, glyphclass2, value2)
+        otValue1 = makeOpenTypeValueRecord(value1, pairPosContext=True)
+        otValue2 = makeOpenTypeValueRecord(value2, pairPosContext=True)
+        lookup.addClassPair(location, glyphclass1, otValue1, glyphclass2, otValue2)
 
     def add_subtable_break(self, location):
         self.cur_lookup_.add_subtable_break(location)
 
     def add_specific_pair_pos(self, location, glyph1, value1, glyph2, value2):
         lookup = self.get_lookup_(location, PairPosBuilder)
-        lookup.addGlyphPair(location, glyph1, value1, glyph2, value2)
+        otValue1 = makeOpenTypeValueRecord(value1, pairPosContext = True)
```

Comment:
pls, no spaces around equal signs in function parameters. I would say "run black autoformatter" but actually I only want it to reformat the modified lines (whereas black reformats everything in the file).